packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -963,7 +1044,9 @@ curl -X GET \
         "address",
         "phone",
         "offline_access",
-        "groups"
+        "groups",
+        "online_access", [Early Access](/docs/reference/releases-at-okta/#early-access-ea)
```

Comment:
We avoid putting any time of comments in request and response examples like this. Can it go at the top of the example and just make note of the what is EA?